sqldelight-core/src/test/kotlin/com/squareup/sqldelight/core/DuplicateSqlIdentifiers.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2017 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.squareup.sqldelight.core
+
+import com.google.common.truth.Truth.assertThat
+import com.squareup.sqldelight.core.util.FixtureCompiler
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+
+class DuplicateSqlIdentifiers {
```

Comment:
Should we do class per failure case or method in `ErrorTests` for failure case?

I think just a method in `ErrorTest` is fine and then abstract things out over time (maybe like a `FunctionErrorTest` which deals only with function names.

Reply:
Guess that really depends on the tests cases, even mixing string-based and file-based ones in a single test file should be fine